mne/epochs.py

```diff
@@ -1233,6 +1242,15 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
+                if any([k_e not in epochs.event_id for k_e in key]):
+                    key = [k for k in epochs.event_id.keys()
+                           if all([set(k_i.split('/')).issubset(k.split("/"))
+                                   for k_i in key])]
+                    if len(key) == 0:
+                        raise KeyError('Attempting selection of events via '
+                                       'multiple/partial matching, but no '
+                                       'event matches all criteria.')
```

Comment:
I don't know if we need to do that -- we can just be explicit in the docs that `a/b` and `b/a` will be treated similarly for sub-selection purposes, and if you want one or the other you need to ask for `a/b` and `b/a` and not use the "tag"-type functionality